website/docs/components/table/advanced-table/partials/guidelines/guidelines.md

```diff
@@ -251,4 +248,14 @@ A multi-select pattern consists of:
 
 2. Row level select allowing for the selection of an individual row.
 
-![](/assets/components/table/advanced-table/table-multi-select-cells.png)
\ No newline at end of file
+![](/assets/components/table/advanced-table/table-multi-select-cells.png)
+
+## Resizable columns
```

Comment:
Have you considered nesting this section within the "Columns" section?

Reply:
Good suggestion, I've updated this.